gremlin-go/README.md

```diff
@@ -65,91 +58,222 @@ import (
 
 func main() {
 	// Creating the connection to the server.
-	driverRemoteConnection := gremlingo.NewDriverRemoteConnection("localhost", 8182)
+	driverRemoteConnection, err := gremlingo.NewDriverRemoteConnection("localhost", 8182)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
 	// Cleanup
-	defer driverRemoteConnection.Close()
-	// Submit a traversal (string format in milestone 1).
+	defer func(driverRemoteConnection *gremlingo.DriverRemoteConnection) {
+		err := driverRemoteConnection.Close()
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+	}(driverRemoteConnection)
```

Comment:
Why did this become this complicated?

Reply:
There is the added error return here, and apparently to defer with error checking, you need to wrap it in a closure. Unless you know of another method?

Reply:
You just ignore the error because you don't care about it when cleaning up.